lib/src/resolver.dart

```diff
@@ -158,9 +158,9 @@ class Loader {
 
   /// Loads an imported resource and returns a [Future] with a [List] of lines.
   /// Returns `null` if the resource could not be loaded.
-  Future<List<String>> load(String path) async {
+  List<String> load(String path) {
     try {
-      return new File(path).readAsLines();
+      return new File(path).readAsLinesSync();
```

Comment:
Since I/O operations are slow, and this is called in a loop over potentially thousands of source files, we want this to be async in order to allow the VM to continue processing lines while we wait for the underlying OS syscalls to load the next source(s).

Reply:
By the way, if you have measured and *are* seeing performance improvements running on source sets with on the order of thousands of files, I'd be very curious to know!